rock_paper_scissors.py

```diff
@@ -103,6 +99,10 @@ def rounds(self, value):
             print('Please enter odd number of rounds.')
             exit()
 
+    @property
+    def valid_choices(self):
+        return list(VALID_CHOICES.keys())
```

Comment:
I like this!

One small suggestion: a tuple would be a more correct container than a list, as tuples (like the choices in this container) are immutable; lists, on the other hand, can be modified.